net-libs/libbitcoinconsensus/libbitcoinconcensus-0.15.1.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Will do later"
+DESCRIPTION="Will do later this is a draft"
+HOMEPAGE="https://bitcoin.org"
+COPYRIGHT="idk"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/bitcoin/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="98b3a98a67054123bef6714e4035e18f9250a1b6d63829258ad057906e75d8b1"
+SOURCE_DIR="bitcoin-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libbitcoinconsensus = $portVersion
+	#"bitcoind" seems to be what
+	#the non GUI bitcoin app is called.
+	app:bitcoind = $portVersion
+	"
+REQUIRES="
+	#I know this should be in alphabetical order...
+	#but I'll do that later
+	haiku$secondaryArchSuffix
+	lib:libminiupnpc$secondaryArchSuffix
+	cmd:univalue$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libevent$secondaryArchSuffix
+	lib:libdb$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libzmq$secondaryArchSuffix
+	cmd:protobuf
```

Comment:
Move the cmd: lines to BUILD_PREREQUIRES

Reply:
Okay.